e2e/tests/ui/features/@sbom-explorer/sbom-explorer.feature

```diff
@@ -63,6 +63,7 @@ Feature: SBOM Explorer - View SBOM details
             | quarkus-bom | jdom        |
 
     Scenario Outline: View SBOM Vulnerabilities
+        Given An ingested SBOM "<sbomName>" is available
         Given An ingested SBOM "<sbomName>" containing Vulnerabilities
```

Comment:
Line 67 was doing 2 things:
- Navigate to the SBOM page and then search for an SBOM
- Verify the vulnerabilities count

Line 66 was added so we reuse the first step of "Navigating to the SBOM page and search for an SBOM"